Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.Source.cs

```diff
@@ -106,6 +106,14 @@ private static bool IsHttpUri(Uri uri)
                 if (IsCacheEnabled && _isHttpSource)
                 {
                     _image.Source = await ImageCache.GetFromCacheAsync(_uri);
+                    if (_image.Source != null)
```

Comment:
Should we signal ImageOpened and ImageFailed here also? I'm not sure what to pass as RoutedEventArgs and ExceptionRoutedEventArgs.